Modules/UserModule/UserModule.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I didn't touch this, but not sure why it shows here.

Reply:
It's ok. Sometimes the diff might no changes but it will show up with this. Just a diff only.
![image](https://github.com/kevinboon3288/cheque-writer/assets/38966644/0c5563a2-0ee6-493d-8344-4b9d1bdc52bd)